### DISCUSSION TITLE
Home assistant irraggiungibile dopo qualche ora

### DISCUSSION THREAD

**Discussion by davide1295** [indomus/forum#1629] [HUB: Home Assistant]
Ciao a tutti, mi sto interfacciando al mondo di home assistant installandolo su un Raspberry Pi 3 model B, scheda sd 16gb classe 4, connessione ethernet.
Sono riuscito ad installarlo, avviarlo e fare qualche esperimento. Tutto sembra funzionare come dovrebbe (devo solo imparare ad usarlo), il problema sorge se non utilizzo il tutto da qualche ora. Ad esempio, la sera riesco ad accedere tramite IP e "homeassistant.local:8123" mentre la mattina se provo ad accedere agli indirizzi mi compare un messaggio di errore non riuscendo ad effettuare il login. L'unico modo per riconnettermi è reinstallare tutto il sistema tramite il software Raspberry Pi imager. Nella pagina del router la mattina trovo il raspberry connesso come la sera precedente.
Come mai questo comportamento?

**Comment by snakuzzo** [comment]
E' impossibile darti una mano senza alcun dettaglio.
Questo comportamento non è normale....se ti si pianta tutto e sei costretto a reinstallare, l'unica cosa che mi viene in mente è un problema con il filesystem (SD diffettosa ?). Se così non fosse sappi che reinstallare non è MAI la soluzione.

**Reply by davide1295** [reply]
Non so proprio cosa fare in quei casi, neanche se riavvio torna a funzionare

**Reply by snakuzzo** [reply]
E' impossibile darti una mano. Mi spiace

**Reply by indomus** [reply]
> Non so proprio cosa fare in quei casi, neanche se riavvio torna a funzionare

Perdonaci, ma come possiamo aiutarti così? :)

**Comment by 50494554524F** [comment]
se stai utilizzando homeassistant os usa la versione 5.11, questa non si pianta...

**Reply by snakuzzo** [reply]
scusa...ma questa cosa è documentata? Perché se così fosse sarebbe pieno di gente con questi problemi

**Reply by 50494554524F** [reply]
non lo so, a me è l'unica che non si blocca... tra qualche giorno che potrò spegnere i termosifoni proverò la 7.xx

**Reply by snakuzzo** [reply]
Sì ma è una pezza basata sul nulla...i problemi non si risolvono così e vi chiediamo cortesemente di non dare consigli del genere basati su un semplice " me è l'unica che non si blocca".
Servono evidenze per affermare certe cose

**Reply by 50494554524F** [reply]
come basata sul nulla?!?! prendi un raspberry e verifica... cosa serve di più?!

**Reply by snakuzzo** [reply]
funziona...

**Reply by snakuzzo** [reply]
e ti ho detto...l'evidenza non è un "a me non funziona".

**Reply by 50494554524F** [reply]
no, quella è una evidenza perché se metto un versione successiva si blocca... poi oh, ognuno perderà tempo come vuole, io non posso mettermi a debuggare homeassistant

**Reply by snakuzzo** [reply]
ok...ma tu puoi fare come ti pare infatti...solo che ti dico che la 7.x non si blocca a prescindere. Poi se hai evidenze, a parte la tua esperienza personale, lieti di dare un'occhiata.
Se l'approccio è "non ho tempo", sappi che è completamente sbagliato. Se hai dei problemi su questi tipi di progetti, si va sul progetto, si segnala e si cerca di capire perché non va...non è che si va a tentoni tornando indietro con le versioni.
Da apprezzare il fatto che tu voglia dare una mano, ma tocca fare le cose per bene.
Poi se a @davide1295 con il "tuo metodo" risolve tutto, fa piacere per voi...ma sappiate che state andando avanti affidandovi al caso

**Reply by 50494554524F** [reply]
finalmente ho installato la 8.1 di haos, è qualche giorno che va senza bloccarsi... potete aggiornare, incrociamo le dita :-D

**Reply by KC-inDomus** [reply]
diciamo che potevamo aggiornare anche senza conferma :))
